### ISSUE TITLE
Fix train_test_split

### ISSUE THREAD

**Issue by gchrupala** [gchrupala/vgs#2]
https://github.com/gchrupala/vgs/blob/8249ff62d2afff5d537126e5a86b23e0698d6325/vg/scorer.py#L125

The splits are not the same if you call train_test_split more than once, even with the same random seed.
